x-pack/plugins/apm/public/components/app/mobile/service_overview/index.tsx

```diff
@@ -193,7 +193,6 @@ export function MobileServiceOverview() {
                     kuery={kueryWithMobileFilters}
                     environment={environment}
                     fixedHeight={true}
-                    isSingleColumn={isSingleColumn}
```

Comment:
`isSingleColumn` in this case is equal to `isLarge` so I removed the extra prop and used `isLarge` directly from the hook